src/collection.js

```diff
@@ -494,6 +494,21 @@ class Collection {
     return this
   }
 
+  /**
+   * Returns median of the current collection
+   *
+   * @param {}
+   *
+   * @returns {Number}
+   */
+  async median () {
+    const arr = (await this.sort()).items
+    const mid = Math.floor(arr.length / 2)
+    return arr.length % 2 !== 0
+      ? arr[mid]
+      : (arr[mid] + arr[(mid - 1)]) / 2

```

Comment:
Also, without `arr` being available, directly access the items using `this.items`

Reply:
Done. Please Check